src/h5web/toolbar/controls/DomainSlider/DomainSlider.tsx

```diff
@@ -72,10 +71,15 @@ function DomainSlider(props: Props): ReactElement {
         step={step}
         disabled={disabled}
         onAfterChange={(bounds) => onCustomDomainChange(bounds as Domain)}
-        renderThumb={(thumbProps, { valueNow }) => (
-          <div {...thumbProps} tabIndex={disabled ? -1 : 0}>
-            <div className={styles.thumbBtnLike}>{formatThumb(valueNow)}</div>
-          </div>
+        renderThumb={({ ref, ...thumbProps }, { index }) => (
+          <Thumb
+            ref={ref as Ref<HTMLDivElement>}
+            index={index}
+            isAutoMin={isAutoMin}
+            isAutoMax={isAutoMax}
+            disabled={disabled}
+            {...thumbProps}
+          />
```

Comment:
Good call but I would suggest to change the `Thumb` component to factor out the `index` logic:

```
        renderThumb={({ ref, ...thumbProps }, { index }) => index == 0 ?
          (<Thumb
            ref={ref as Ref<HTMLDivElement>}
            isAuto={isAutoMin}
            icon={FiSkipBack}
            disabled={disabled}
            {...thumbProps}
          />)
:          (<Thumb
            ref={ref as Ref<HTMLDivElement>}
            isAuto={isAutoMax}
            icon={FiSkipForward}
            disabled={disabled}
            {...thumbProps}
          />)}
```

I decline responsibility in case of unmatched braces

Reply:
The idea was to lighten up `DomainSlider`. Isn't this counter-productive? I'd much prefer the complexity to be in `Thumb`.

Reply:
You're right, though. Instead of rendering two thumbs in a ternary, I'll render one thumb with ternaries on a couple of props.

Reply:
I see.

My opinion is that while it allows to write less lines in `DomainSlider`, it is at the cost of unneeded complexity in `Thumb`:
- You need to pass two props `isAuto..`
- You need to check which one is the correct one according to the index
- You need to choose which Icon to display according to the above check AND to the index

All these checks can be factored out, making the `Thumb` component and its call way easier to understand in my opinion.

A compromise would be to keep the signature of `Thumb` but to have two branches in it:
```
if (index === 0) {
  # return thumbLeft
}
else {
  # return thumbRight
}
```

Anyway, these are suggestions, feel free to disregard if you don't agree :slightly_smiling_face: